src/main/java/org/myrobotlab/framework/repo/IvyWrapper.java

```diff
@@ -436,8 +437,24 @@ synchronized public void install(String location, String[] serviceTypes) {
 
       Platform platform = Platform.getLocalInstance();
 
-      // templates [originalname](-[classifier])(-[revision]).[ext]  parens are "optional"
-      String[] cmd = new String[] { "-settings", location + "/ivysettings.xml", "-ivy", location + "/ivy.xml", "-retrieve", location + "/jar" + "/[originalname].[ext]" };
+      // templates [originalname](-[classifier])(-[revision]).[ext] parens are
+      // "optional"
+
+      List<String> cmd = new ArrayList<>();
+      cmd.add("-settings");
+      cmd.add(location + "/ivysettings.xml");
+      cmd.add("-ivy");
+      cmd.add(location + "/ivy.xml");
+      cmd.add("-retrieve");
+      cmd.add(location + "/jar" + "/[originalname].[ext]");

```

Comment:
Should probably be using `fs` for file separator, technically the `/` works on Windows, it's just not advised.

Reply:
kinda adverse to add more clutter .. it would need to be Service.fs